### ISSUE TITLE
Training strategy

### ISSUE THREAD

**Issue by haoyuhsu** [FurkanOM/tf-faster-rcnn#13]
Hi, I'm now facing a problem when training on your code.
I have trained about 37 epochs on PASCAL VOC 2007 trainval dataset directly, and it turns out no bounding box appeared on test image.
I wonder what is your training procedure, for example,  changing learning rate value or training RPN and Regression layers alternatively.
Thanks a lot!!!

**Comment by AlessandroZoia-dev** [comment]
Can I ask you informations about your training strategy and mAp obtained after the training?
Thank you very much!
